packages/algob/test/internal/util/scripts-runner.ts

```diff
@@ -25,22 +25,22 @@ describe("Scripts runner", function () {
 	});
 
 	it("Should pass params to the script", async function () {
-		await runScript("./scripts/params-script.js", env, new DeployerRunMode(deployerCfg));
+		await runScript("./scripts/params-script.js", [], env, new DeployerRunMode(deployerCfg));
```

Comment:
Could you test with an argument pass in the script?

Reply:
Done